Misc/NEWS.d/next/Library/2019-03-27-02-09-22.bpo-36385.we2F45.rst

```diff
@@ -0,0 +1 @@
+Stop rejecting IPv4 octets for leading zeroes.
```

Comment:
I think we can make this more clear. Maybe "No longer reject IPv4 octets that begin with leading zeros. Leading zeros are ignored, and no longer are assumed to specify octal octets. Octets are always decimal numbers. The requirement that octets be no more than three digits, including leading zeros, is still enforced."

Reply:
Expanded the description a little.